src/ui/atoms/label-control/index.tsx

```diff
@@ -0,0 +1,21 @@
+import styled, { StyledComponent } from 'styled-components';
+
+interface LabelControlProps {
+  text: React.ReactNode;
+  children?: never;
+}
+
+const map = (properties: LabelControlProps) => ({
+  children: properties.text,
+});
+
+export const Base = styled.span.attrs(map)`
+  font-size: var(--font-size);
+  line-height: var(--line-height);
+  padding: var(--padding);
```

Comment:
Добавляй дефолтные значения во время чтения переменных.

```suggestion
  padding: var(--padding, 1rem);
```